src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionUIConfig.java

```diff
@@ -79,10 +79,9 @@ public BlackDuckDistributionUIConfig(final BlackDuckProvider provider) {
         final String projectNamePattern = fieldModel.getField(BlackDuckDescriptor.KEY_PROJECT_NAME_PATTERN).flatMap(field -> field.getValue()).orElse(null);
         final boolean trueTextPresent = Boolean.TRUE.toString().equalsIgnoreCase(filterByProject);
         final boolean falseTextPresent = Boolean.FALSE.toString().equalsIgnoreCase(filterByProject);
-        if (!trueTextPresent && !falseTextPresent) {
-            if ((null == configuredProjects || configuredProjects.isEmpty()) && StringUtils.isBlank(projectNamePattern)) {
-                return List.of("You must select at least one project.");
-            }
+        final boolean missingProject = (null == configuredProjects || configuredProjects.isEmpty()) && StringUtils.isBlank(projectNamePattern);
+        if (!trueTextPresent && !falseTextPresent && missingProject) {
```

Comment:
Negating both the true and false case looks like a bug. I know you didn't necessarily write the original logic, but we should be using `final boolean shouldFilter = Boolean.parseBoolean(filterByProject)` to get the value, then using that for our logic. Anything else kills readability.